processing-and-analysis/02-command-files/2s-practice-jackson/mlm-work-unique-ta-sects-jh.Rmd

```diff
@@ -58,140 +58,33 @@ master_unique <- na.omit(master_unique_2)
 
 ### Make `ta_sect` values unique across `course_fullid`
 
-#### First, I'm going to temporarily change `ta_sect` into character values
-
-```{r}
-master_unique$ta_sect <- as.character(master_unique$ta_sect)
+#### I'm going to be trying the code that CFC showed us
+
+```{r, eval = FALSE}
+master_unique_3 <- master_unique %>% 
+  select(course_fullid)
+master_unique_4 <- master_unique_3 %>% 
+  mutate(ta_sect = str_extract(course_fullid, "\\d\\d\\d\\d")) "\\d{4}"
+#### This line ^^ is the one that is giving me trouble
+master_true <- master_unique_3 %>% 
+  mutate(ta_sect = ifelse(str_detect(course_fullid, "2016"), paste(ta_sect, "16", sep = "_"), paste(ta_sect, "17", sep = "_")))
```

Comment:
I don't know why this wouldn't work...nothing jumps out at me as an error. What sort of message did you get when it failed?